ansible/install-letsencrypt.yml

```diff
@@ -3,10 +3,11 @@
   vars:
     ansible_python_interpreter: "/usr/bin/python"
   roles:
+    - common
     - docker_py
 
 - hosts: local:custom
   vars:
-    ansible_python_interpreter: "{{ ansible_env.HOME }}/venv/bin/python2.7"
+    ansible_python_interpreter: "/root/venv/bin/python2.7"
```

Comment:
We should find a better way to do this.
In my case (ansible version 2.0.2.0) `ansible_env.HOME` is not expanding correctly.